templates/firefox-ios.xliff

```diff
@@ -1930,16 +1967,36 @@
         <source>You can choose which list Firefox will use to block Web elements that may track your browsing activity.</source>
         <note>Header description for tracking protection options section where Basic/Strict block list can be selected</note>
       </trans-unit>
-      <trans-unit id="Settings.TrackingProtection.BlockListsTitle">
-        <source>Block Lists</source>
-        <note>Title for tracking protection options section where Basic/Strict block list can be selected</note>
+      <trans-unit id="Settings.TrackingProtection.Info.BasicTitle">
+        <source>Balanced for protection and performance.</source>
+        <note>Explanation of basic mode.</note>
+      </trans-unit>
+      <trans-unit id="Settings.TrackingProtection.Info.BlocksTitle">
+        <source>BLOCKS</source>
+        <note>Title on info view.</note>
```

Comment:
It's confusing to me what "BLOCKS" refers to here. Could you please give a bit more context in the localization comment?